src/components/Tabs.react.js

```diff
@@ -0,0 +1,204 @@
+          : `${className || ''} tab`
+      }
+      style={selected ? { ...style, ...selectedStyle } : style}
+      onClick={() => {
+        selectHandler(index);
+      }}
+    >
+      <span>{label}</span>
+      <style jsx>{`
+        .tab {
+          display: inline-block;
+          background-color: ${offWhiteColor};
+          border: 1px solid ${greyColor};
+          padding: 20px;
+          transition: background-color, color 200ms;
+          font-family: 'system-ui';
+          width: 100%;
+          box-sizing: border-box;
+        }
+        .tab:hover {
+          cursor: pointer;
+        }
+        .tab--selected {
+          border-top: 2px solid ${primaryColor};
+          border-bottom: none;
+          color: black;
+        }
+
+        @media screen and (min-width: 1000px) {
+          .tab {
+            width: auto;
+          }
+        }
+      `}</style>
+    </div>
+  );
+};
+
+export default class Tabs extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      selected: 0
+    };
+
+    this.selectHandler = this.selectHandler.bind(this);
+  }
+  selectHandler(index) {
+    this.setState({
+      selected: index
+    });
+  }
+  render() {
+    // enhance Tab components coming from Dash (as dcc.Tab) with methods needed for handling logic
+    // TODO: handle components that are not dcc.Tab components (throw error)
+    const EnhancedTabs = this.props.children.map((child, index) => {
+      return (
+        <EnhancedTab
+          key={index}
+          index={index}
+          label={child.props.label || child.props.children.props.label}
+          selected={this.state.selected === index}
+          selectHandler={this.selectHandler}
+          children={child.props.children}
+          className={child.props.className}
+          style={child.props.style}
+          selectedClassName={child.props.selectedClassName}
+          selectedStyle={child.props.selectedStyle}
+        />
+      );
+    });
+
+    const tabContainerClass = this.props.vertical
+      ? 'tab-container tab-container--vert'
+      : 'tab-container';
+
+    const tabContentClass = this.props.vertical
+      ? 'tab-content tab-content--vert'
+      : 'tab-content';
+
+    const tabParentClass = this.props.vertical
+      ? 'tab-parent tab-parent--vert'
+      : 'tab-parent';
+
+    return (
+      <div
+        className={`${tabParentClass} ${this.props.parentClassName || ''}`}
+        style={this.props.parentStyle}
+      >
+        <div
+          className={`${tabContainerClass} ${this.props.className || ''}`}
+          style={this.props.style}
+        >
+          {EnhancedTabs}
+        </div>
+        <div
+          className={`${tabContentClass} ${this.props.contentClassName || ''}`}
+          style={this.props.contentStyle}
+        >
+          {this.props.children[this.state.selected].props.children}
+        </div>
+        <style jsx>{`
+          .tab-parent {
+            display: flex;
```

Comment:
We try to support IE11 and unfortunately `flex` is only partially supported: https://caniuse.com/#feat=flexbox

We have a browserstack account that you can use to check if this subset of `flex` features are compatible with `IE11`